src/components/MainPage/Importers/Importer.vue

```diff
@@ -1,52 +1,70 @@
 <template>
-  <div v-if="decryptedImporter">
-    <div>
-      <el-checkbox v-model="showBrowser">
-        Show browser
-      </el-checkbox>
-    </div>
-    <div
-      v-for="(value, loginField) in decryptedImporter.loginFields"
-      :key="loginField"
-    >
-      {{ loginField }}:
-      {{ loginField != "password" ? value : "*".repeat(value.length) }}
-    </div>
-    <el-button
-      type="primary"
-      :loading="importing"
-      @click="scrape()"
-    >
-      Import
-    </el-button>
-    <el-button
-      type="danger"
-      icon="el-icon-delete"
-      :disabled="importing"
-      @click="promptDelete()"
-    >
-      Delete
-    </el-button>
-    <el-progress
-      v-show="importing"
-      :percentage="percentage"
-      :status="status"
-      class="progress-bar"
+  <div>
+    <v-container v-if="decryptedImporter">
+      <v-checkbox
+        v-model="showBrowser"
+        :label="`Show browser`"
+      />
+      <div
+        v-for="(value, loginField) in decryptedImporter.loginFields"
+        :key="loginField"
+      >
+        {{ loginField }}:
+        {{ loginField != "password" ? value : "*".repeat(value.length) }}
+      </div>
+      <v-btn
+        color="primary"
+        class="mx-1"
+        :loading="importing"
+        @click="scrape()"
+      >
+        Import
+      </v-btn>
+      <v-btn
+        color="error"
+        dark
+        class="mx-1"
+        @click="deleteDialog = true"
+      >
+        <v-icon>mdi-delete</v-icon>
+        Delete
+      </v-btn>
+      <DeleteImporterDialog
+        v-model="deleteDialog"
+        @confirm="DeleteImporter"
+      />
+      <v-progress-linear
+        v-show="importing"
+        v-model="percentage"
+        height="25"
+        reactive
+      >
+        <template v-slot="{ value }">
+          <strong>{{ value }}%</strong>
+        </template>
+      </v-progress-linear>
+      <div v-show="importing">
+        {{ step }}
+      </div>
+    </v-container>
+    <v-skeleton-loader
+      v-else
+      type="paragraph"
     />
```

Comment:
![image](https://user-images.githubusercontent.com/17686879/75475350-52ae9000-59a1-11ea-8def-8ba425d995d9.png)

Reply:
How did you catch this image
it is only takes something like 20-50 milliseconds to decrypt isn't

Reply:
I think it is good practice to show **something** with the `v-else`, you don't know when will be a bug in decryption.

For the screenshot I set `v-if="false"` in the source code.